docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,80 @@
+In this tutorial, we'll walk you through the process of deploying the indico charm, relating it to the nginx-ingress-integrator charm, the postgresql-k8s charm and the redis-k8s charm, and inspecting kubernetes resources it creates. We'll then also look at changing the charm configuration via a juju configuration update.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+
+## Deploy this charm
+
+Discourse requires connections to PostgreSQL and Redis, so those will be deployed too and related to the Discourse charm. For more information, see the [Charm Architecture](https://charmhub.io/discourse-k8s/docs/charm-architecture).
+
+All the above charm will the deployed in a new model names `discourse`:
+
+```
+# Add the model
+juju add-model discourse
+
+# Deploy the charms
+juju deploy redis-k8s
+juju deploy postgresql-k8s
+juju deploy discourse-k8s
+
+# Relate redis-k8s and postgresql-k8s to discourse-k8s
+juju relate redis-k8s discourse-k8s
+# For postgresql-k8s the interface db needs to be specified as the charm provides more than one
```

Comment:
I'd put quotes around "db" and put it before "interface" as it'll read a little better that way:
```
For postgresql-k8s the "db" interface needs to be specified as the charm provides more than one
```

Reply:
Fixed